frontend/packages/patternfly/src/components/notification-drawer/notification-drawer-heading.tsx

```diff
@@ -11,19 +11,19 @@ const NotificationDrawerHeading: React.FC<NotificationDrawerHeadingProps> = ({
     <div className="pf-c-notification-drawer">
       <div className="pf-c-notification-drawer__header">
         <h1 className="pf-c-notification-drawer__header-title">
-          {t('notification-drawer~Notifications')}
+          {t('public~Notifications')}
```

Comment:
Messages under the packages directory shouldn't go into the `public` i18n namespace. We have a separate namespace for each package.